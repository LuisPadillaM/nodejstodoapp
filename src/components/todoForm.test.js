import React from 'react';
import renderer from 'react-test-renderer';
import TodoForm from './todoForm.js';


test('TodoForm render test', () => {

  const component = renderer.create(
    <TodoForm add={(x) => x} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

