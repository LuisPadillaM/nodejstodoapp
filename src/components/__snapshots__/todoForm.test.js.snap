// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoForm render test 1`] = `
<form
  action=""
  method="POST"
  onSubmit={[Function]}
>
  <div
    className="form-first"
  >
    <div
      className="input_field"
      data-react-toolbox="input"
    >
      <input
        className=""
        defaultValue={undefined}
        disabled={false}
        maxLength={undefined}
        name={undefined}
        onChange={[Function]}
        onKeyPress={undefined}
        required={true}
        role="input"
        type="text"
        value=""
      />
      <span
        className={undefined}
      />
      <label
        className=""
      >
        New item
        <span
          className={undefined}
        >
           * 
        </span>
      </label>
    </div>
    <button
      className=""
      data-react-toolbox="button"
      disabled={false}
      href={undefined}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchStart={[Function]}
      type="submit"
    >
      <span
        aria-label=""
        className="material-icons"
        data-react-toolbox="font-icon"
      >
        add
      </span>
    </button>
  </div>
</form>
`;
